### ISSUE TITLE
Use releases

### ISSUE THREAD

**Issue by MaxGhenis** [mwburke/stargazer#50]
Especially if considering backward-incompatible changes like #44.

**Comment by toobaz** [comment]
Do you mean github releases instead than/in addition to pypi releases?

**Comment by MaxGhenis** [comment]
Yes, though if there's a way to document pypi releases on github that could work too I suppose (I didn't see release notes on the [page](https://pypi.org/project/stargazer/#history)). Most projects I've seen use github releases though.
